packages/sagas/src/pending/pending.js

```diff
@@ -0,0 +1,62 @@
+import { delay } from 'redux-saga';
+import { call, put, select, take } from 'redux-saga/effects';
+import { api } from '@talend/react-cmf';
+import { Map } from 'immutable';
+
+import { PENDING_DELAY_TO_SHOW, SHOW_PENDING } from '../constants';
+
+const addOrReplace = api.actions.collections.addOrReplace;
+
+/**
+ * find a datastore by its id
+ * @param {*} state the application state
+ * @param {*} id the datastoreId
+ */
+export function findPenderById(state, asyncActionId) {
+	return state.cmf.collections.getIn(['penders', asyncActionId]);
```

Comment:
Shouldn't `penders` be a shared contant?

Reply:
I'm not even sure `penders` means anything :thinking:

Reply:
Strict translation does not mean the right stuff, but you do get the right meaning.
pendingComponents would maybe be more accurate.